packages/scaffold-ui/src/partials/w3m-header/index.ts

```diff
@@ -213,6 +235,21 @@ export class W3mHeader extends LitElement {
     ></wui-icon-link>`
   }
 
+  private onNetworks() {
+    if (this.isAllowedNetworkSwitch()) {
+      EventsController.sendEvent({ type: 'track', event: 'CLICK_NETWORKS' })
+      RouterController.push('Networks')
+    }
+  }
+
+  private isAllowedNetworkSwitch() {
```

Comment:
Do we use this?

Reply:
This is quite old logic and should be considered separately. Noted.

Reply:
This is quite old logic and shouldn't be responsibility of this PR. Noted to refactor later.